drivers/sensor/iis2iclx/iis2iclx.h

```diff
@@ -15,6 +15,7 @@
 #include <zephyr/types.h>
 #include <drivers/gpio.h>
 #include <sys/util.h>
+#include "../stmemsc/stmemsc.h"
```

Comment:
What about moving `stmemsc.h` to `include/drivers/sensor`and include it from there ?

Reply:
ok, it seems this was the initial proposal. Though I'm not sure to understand why this is not a valid solution

Reply:
Correct, it was the initial proposal. To me was the best solution. In that case no need to add an include path.

Reply:
you should be able to do:
```suggestion
#include <stmemsc.h>
```

and then in: https://github.com/zephyrproject-rtos/zephyr/blob/master/drivers/sensor/iis2iclx/CMakeLists.txt
```
zephyr_library_include_directories(../stmemsc)
```

Reply:
OK, I will do like that. Thanks for suggestion.